weasyl/middleware.py

```diff
@@ -19,6 +23,12 @@
 from weasyl.error import WeasylError
 
 
+def _get_headers(env):
+    return {
+        key[5:].replace('_', '-').title(): value
+        for key, value in env.iteritems() if key.startswith('HTTP_')}
```

Comment:
Would you hate me too much for suggesting putting it in the `web.py` fork?

Trying to prevent enabling people who think `web.py` is a good idea aside, maybe `weasyl.compat` or `weasyl.dry`. There's a few dumping grounds for things like this. (Maybe worth a cleanup ticket? Not to sidetrack this too much.)